VocaDbWeb/Views/Album/Details.cshtml

```diff
@@ -16,6 +16,8 @@
 @{
 	ViewBag.Parents = new[] { Html.ActionLink(ViewRes.SharedStrings.Albums, "Index") };
 	var url = PageProperties.CanonicalUrl;
+
+	PageProperties.Robots = Model.Deleted ? "noindex,follow" : string.Empty;

```

Comment:
Can you add a named constant for these?

Reply:
What would you name those constants? We have two different values, `noindex,follow` and `noindex,nofollow`. And where would you place them?

Reply:
Maybe you could place them in the PagePropertiesData class? The name could be just ```Robots_NoIndex_Follow``` (I think it's still better to have named constant, even if the constant includes the whole string)